### PR TITLE
pfblockerng-devel: Add feeds from firebog.net. Implements #11022.

### PR DESCRIPTION
**Pull request by Gosherm**
Add additional blocker feeds listed at firebog.net.

https://redmine.pfsense.org/issues/11022

### CODE REVIEW

**Comment by rbgarga** [comment]
@BBcan177 can you please take a look?

**Comment by Gosherm** [comment]
@BBcan177, @rbgarga - This PR has been updated to be ready for merging with pfblockerng-devel 3

File: net/pfSense-pkg-pfBlockerNG-devel/Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 PORTNAME=	pfSense-pkg-pfBlockerNG-devel
 PORTVERSION=	3.0.0
-PORTREVISION=	2
+PORTVERSION=
```
Comment (rbgarga) [inline net/pfSense-pkg-pfBlockerNG-devel/Makefile:5]: You must set it to 3 instead of empty

**Review by rbgarga — CHANGES_REQUESTED** [review]

**Comment by rbgarga** [comment]
@BBcan177 can you review this one please?

**Comment by BBcan177** [comment]
@Gosherm 

For now, I would like to leave Malware Domain Lists in the Feeds.

Also can you remove the commit to the makefile, as I have some pending changes to make to the code, so I want to minimize the number of installation notifications that the users get.

Otherwise OK to merge.

Thanks!

File: net/pfSense-pkg-pfBlockerNG-devel/Makefile (hunk 1 above)
Comment (Gosherm) [reply net/pfSense-pkg-pfBlockerNG-devel/Makefile]: Set back to 2 per request from BBCAN177

**Comment by Gosherm** [comment]
Reverted makefile and malwarebytes changes per requests

**Comment by rbgarga** [comment]
@BBcan177 please review

**Comment by BBcan177** [comment]
@Gosherm 

The **Info/Description** attributes should be more informative. 

Can you add the  "**info**" attribute to all of the feeds which are hosted on https://v.firebog.net/ which are formatted copies of the original source of those feeds, to indicate that to the users as info.

Otherwise its OK, but will get this merged once I submit the next version.

**Comment by rbgarga** [comment]
@Gosherm when you address the change asked by @BBcan177 please rebase your fork and fix conflicts

**Comment by Gosherm** [comment]
@rbgarga Rebased and unconflicted.

**Comment by Gosherm** [comment]
@BBcan177 Changes made as requested

**Comment by BBcan177** [comment]
@Gosherm 
I just added a PR #1008 which is hopefully approved and merged tomorrow.

Unfortunately, the JSON for this PR is invalid. You need to add a "," to the line before each of the new "info:" lines that were recently added. I also suggest that once you make those changes, download the RAW file and test to make sure that the Feeds tab reads the file.

Thanks for your patience, but just need to do the last fixup.

**Comment by BBcan177** [comment]
Hi @Gosherm , would you be able to make these changes as I have another update coming soon. Thanks!

**Comment by BBcan177** [comment]
Hi @Gosherm, just checking to see if you are able to make those changes, or if its easier, I can add this to my next version? Thanks!

**Comment by rbgarga** [comment]
Superseded by #1035

**Pull request closed without merging**